<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QMainWindow" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>406</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>WeatherApp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/images/weather_states/sunny.png</normaloff>:/images/weather_states/sunny.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="WeatherAppTitle">
      <property name="text">
       <string>Weather App</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="cityInfoWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="cityInfoVBoxLayout">
         <item>
          <widget class="QLabel" name="cityNamingLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Texas, USA</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignJustify</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cityCoordinatesLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>50.123NE, 29.123SW</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="currentObservingDateVBoxLayout">
         <item alignment="Qt::AlignRight|Qt::AlignVCenter">
          <widget class="QLabel" name="currentObservingDateLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Current observing date</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QDateEdit" name="currentObservingDateEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="displayFormat">
            <string>d/M/yy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QScrollArea" name="weatherInfoScrollArea">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="daysWeatherWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>863</width>
         <height>213</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="dayVBoxLayout_1">
          <item>
           <widget class="QLabel" name="dayDateLabel_1">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout"/>
          </item>
          <item>
           <widget class="QLabel" name="dayImageLabel_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="daySpacer_1">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dayStubLabel_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>WeatherStub 
Temp: 300K
Pressure: 1100 hPa
Humidity: 80%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dayVBoxLayout_2">
          <item>
           <widget class="QLabel" name="dayDateLabel_2">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dayImageLabel_2">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="daySpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dayStubLabel_2">
            <property name="text">
             <string>WeatherStub 
Temp: 300K
Pressure: 1100 hPa
Humidity: 80%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dayVBoxLayout_3">
          <item>
           <widget class="QLabel" name="dayDateLabel_3">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dayImageLabel_3">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="daySpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dayStubLabel_3">
            <property name="text">
             <string>WeatherStub 
Temp: 300K
Pressure: 1100 hPa
Humidity: 80%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dayVBoxLayout_4">
          <item>
           <widget class="QLabel" name="dayDateLabel_4">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dayImageLabel_4">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="daySpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dayStubLabel_4">
            <property name="text">
             <string>WeatherStub 
Temp: 300K
Pressure: 1100 hPa
Humidity: 80%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dayVBoxLayout_5">
          <item>
           <widget class="QLabel" name="dayDateLabel_5">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dayImageLabel_5">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="daySpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dayStubLabel_5">
            <property name="text">
             <string>WeatherStub 
Temp: 300K
Pressure: 1100 hPa
Humidity: 80%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dayVBoxLayout_6">
          <item>
           <widget class="QLabel" name="dayDateLabel_6">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dayImageLabel_6">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="daySpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dayStubLabel_6">
            <property name="text">
             <string>WeatherStub 
Temp: 300K
Pressure: 1100 hPa
Humidity: 80%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dayVBoxLayout_7">
          <item>
           <widget class="QLabel" name="dayDateLabel_7">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dayImageLabel_7">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="daySpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dayStubLabel_7">
            <property name="text">
             <string>WeatherStub 
Temp: 300K
Pressure: 1100 hPa
Humidity: 80%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_City"/>
    <addaction name="actionOpen_City"/>
    <addaction name="actionNew_Weather_Record"/>
    <addaction name="actionClose_city"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionLoad_weather_graph"/>
    <addaction name="actionWarmest_and_Coldest_days"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_this_project"/>
    <addaction name="actionAbout_author"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_City">
   <property name="icon">
    <iconset theme="folder-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New City...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_City">
   <property name="icon">
    <iconset theme="folder-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open City...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew_Weather_Record">
   <property name="text">
    <string>New Weather Record...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionClose_city">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Close city</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_weather_graph">
   <property name="icon">
    <iconset theme="applications-graphics">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Load weather graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAbout_this_project">
   <property name="icon">
    <iconset theme="help-contents">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About this project</string>
   </property>
  </action>
  <action name="actionAbout_author">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About author</string>
   </property>
  </action>
  <action name="actionWarmest_and_Coldest_days">
   <property name="text">
    <string>Warmest and Coldest days</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
